co-log-core/CHANGELOG.md

```diff
@@ -4,19 +4,20 @@ Change log
 `co-log-core` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
-0.1.1
+Unreleased: 0.1.1
```

Comment:
I like having this tag in the CHANGELOG. Can't count the number of times I lost track of whether or not we had already released the current working version or not :laughing: